ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -108,10 +109,15 @@ If.dmic {
 			Type ControlExists
 			Control "name='Dmic0 Capture Volume'"
 		}
-		True.BootSequence [
-			cset "name='Dmic0 Capture Volume' 70%"
-			cset "name='Dmic0 Capture Switch' on"
-		]
+		True {
+			BootSequence [
+				cset "name='Dmic0 Capture Volume' 70%"
+				cset "name='Dmic0 Capture Switch' on"
+			]
+			Macro.Dmic0.SofEqBootSetup "endpoint='Dmic0' direction='Capture' eqtype='Iir'"
+			Macro.Dmic0.SofDrcBootSetup "endpoint='Dmic0' direction='Capture'"
+			Macro.Dmic0.SofBeamformerBootSetup "endpoint=Dmic0 direction=Capture"
```

Comment:
Nitpicking: Use 'STR' everywhere or remove '' everywhere.

Reply:
I'm not sure what to do, please help me to start. Can you point me an example .conf in UCM2 to follow for quotes style.  Also, is https://www.alsa-project.org/alsa-doc/alsa-lib/group__ucm__conf.html up-to-date for using 'x' and "y"?

Reply:
Ah, above Capture should be 'Capture' ... I'll see if I have similar missed ''.

Reply:
Fixed, I was blind.